classes/helpers/order.php

```diff
@@ -91,9 +92,23 @@ public static function create( $order ) {
 
 		$payment->setMerchantUrls( $merchant_urls );
 
-		$types = array( 'line_item', 'shipping', 'fee', 'coupon' );
+		foreach ( self::get_line_items( $order ) as $line_item ) {
+			$payment->addLineItem( $line_item );
+		}
 
-		foreach ( $order->get_items( $types ) as $item ) {
+		return $payment;
+	}
+
+	/**
+	 * Get the line items for the order.
+	 *
+	 * @param WC_Order|WC_Order_Refund $order The order to get the line items for.
+	 * @return array<LineItem>
+	 */
+	public static function get_line_items( $order ) {
+		$line_items = array();
+
+		foreach ( $order->get_items( array( 'line_item', 'shipping', 'fee', 'coupon' ) ) as $item ) {
```

Comment:
I know in the branch develop-logging we have changed to not include the coupons at all [here](https://github.com/krokedil/zaver-woocommerce-plugin/blob/develop-logging/classes/helpers/order.php#L112). Should this PR maybe be to that branch to prevent any potential issues with conflicts causing a solved bug to reoccur?